CHANGELOG.md

```diff
@@ -1,12 +1,20 @@
 # Changelog
 
-## `0.3.9`
+## `0.4.0`
 
-### `ipyforcegraph 0.3.9`
+### `ipyforcegraph 0.4.0`
 
-> TBD
+- adds `CaptureAs`, which can wrap a `Nunjucks` or `ContinuousColor` to store the
```

Comment:
It'd be dumb to do, but we can also wrap `Column` since it is a `DynamicValue`?

Reply:
yeah, might already work. would add test.

Reply:
actually, the `Color` stuff is _already_ a `Column` subclass... maybe i just add to the example.

Reply:
is now under test :test_tube: , updated text